src/icons/README.md

```diff
@@ -1,113 +1,56 @@
 
-#### As a React component (recommended)
+#### Requesting icons be to FDS (for CB Insights employees)
 
-```
-import CloudIcon from '@cbinsights/fds/lib/icons/react/CloudIcon';
-
-const UploadButton = () => {
-  <button>
-    <CloudIcon />
-    Upload yourself to the cloud
-  </button>
-};
-```
-
-Every Icon component from `fds-icons` accepts the following props:
-
-| Prop        | Type                  | Default          | Description                 |
-| ----------- | --------------------- | ---------------- | --------------------------- |
-| `color`     | `String`              | `COLOR_CHARCOAL` | Color of `fill` for SVG     |
-| `size`      | `xs`,`s`,`m`,`l`,`xl` | `s`              | Standard media size of icon |
-| `className` | `String`              |                  | custom class name           |
-
-#### As raw SVG
-
-```
-import cloudSvg from '@cbinsights/fds/lib/icons/svg/Cloud.svg';
-```
-
-#### As raw PNG
+Most of you will want to request the icons be added to FDS rather than doing it yourselves:
 
-FDS provides PNGs scaled to 1x, 2x, 3x, and 4x.
-The base icon name is scaled to 1x. Specify higher res images with the `@scale` suffix:
+- Verify that the icon you need is in Abstract first: https://app.abstract.com/projects/0b3bf318-e4a9-4319-8a9f-52f161094fb2/branches/master/files/D109C042-4BC3-434D-844A-6F3AA9E2881C
+- Request that we re-run our icon export script in this channel: https://cbinsights.slack.com/archives/CL99MD596
 
-- `Cloud.png` (1x)
-- `Cloud@2x.png` (2x)
+#### Actually adding Icons to FDS
 
-```
-import cloudPng from '@cbinsights/fds/lib/icons/png/Cloud@2x.png';
-```
-
----
-
-## Development
-
-### Updating icons from a Sketch file
+To regenerate the icons, you must
 
-This project uses `sketchtool` to export SVG icons directly from a sketch file provided by the design team. You must have [Sketch](https://www.sketchapp.com/) installed in order to run the export command.
-
-#### 1) Setting up `sketchtool`
-
-If you don't already have `sketchtool` on your system, follow these steps:
-
-1. Install [Sketch](https://www.sketchapp.com/)
-2. Add the `sketchtool` binary to your PATH:
+1. Be a CB Insights employee and request access to this
+   [Abstract Project](https://app.goabstract.com/organizations/0bd48624-8826-4447-a082-1957932b89b8/projects)
+2. Ensure you have [Sketch](https://www.sketchapp.com/) installed.
+3. Ensure the `sketchtool` binary is in your PATH:
 
 ```bash
+## bash
 export SKETCHTOOL_PATH="/Applications/Sketch.app/Contents/Resources/sketchtool/bin"
 export PATH=$PATH:SKETCHTOOL_PATH
+## fish
+set -U /Applications/Sketch.app/Contents/Resources/sketchtool/bin $fish_user_paths
 ```
 
-#### 2) Getting the icons sketch file
+4. Ensure you have the [Abstract Desktop App](https://app.abstract.com/download)
+5. In Abstract Desktop app in the sidebar, select **FDS** --> **Master** in the sidebar
```

Comment:
Clicking master doesn't work if the FDS part isn't already clicked on / focused.